blueprints/addon/files/.travis.yml

```diff
@@ -37,4 +37,4 @@ install:
 script:
   # Usually, it's ok to finish the test scenario without reverting
   #  to the addon's original dependency state, skipping "cleanup".
-  - ember try:one $EMBER_TRY_SCENARIO test --skip-cleanup
+  - node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO test --skip-cleanup
```

Comment:
I agree with both cases actually.  

For a hot-fix in the release branch we should do the smallest possible change, which means changing this file directly to use `node_modules/.bin/ember` here. 

We can totally address a `package.json` `script` based system later in beta/canary...